integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/ReplaceKeys.feature

```diff
@@ -17,14 +16,36 @@ Feature: SmartMetering Configuration - Replace Keys
     Then the replace keys response should be returned
       | DeviceIdentification | TEST1024000000001 |
       | Result               | OK                |
-    And the new keys are stored in the osgp_adapter_protocol_dlms database security_key table
+    And the new keys are stored in the secret management database encrypted_secret table
     And the stored keys are not equal to the received keys
 
   @ResetKeysOnDevice
```

Comment:
Are these Key specific annotations intentionally?

Reply:
Yes, to be able to test key replacement specifically